golem_messages/message/tasks.py

```diff
@@ -561,8 +561,16 @@ def __init__(self, conn_id=None, **kwargs):
 
 
 @library.register(TASK_MSG_BASE + 25)
-class WaitingForResults(base.Message):
-    __slots__ = base.Message.__slots__
+class WaitingForResults(TaskMessage):
+    TASK_ID_PROVIDERS = ('task_to_compute', )
+    EXPECTED_OWNERS = (TaskMessage.OWNER_CHOICES.requestor, )
+    MSG_SLOTS = {
+        'task_to_compute': base.MessageSlotDefinition(TaskToCompute),
```

Comment:
Isn't `WaitingForResults` a reply to `WantToComputeTask`? Wouldn't it be clearer to include wtct instead?

Reply:
Then you won't get information for which task results requestor is waiting.

Reply:
Then maybe include `ReportComputedTask` ?

Reply:
:) @shadeofblue edited #320 this morning and changed rtc to ttc. RTC won't always be available. WFR can happen anytime since subtask assignment.